ingestion/src/kldata.rs

```diff
@@ -336,11 +332,7 @@ pub async fn filter_and_label_kldata<'a>(
                 }
             };
 
-            let (sensor, lvl) = in_datum
-                .id
-                .sensor_and_level
-                .map(|both| (Some(both.0), Some(both.1)))
-                .unwrap_or((None, None));
+            let (sensor, lvl): (i32, i32) = in_datum.id.sensor_and_level.unwrap_or((0, 0));
 
```

Comment:
So we are defaulting ourselves to 0,0, does this make the most sense or could it be handled differently? Are these always both not found? Couldn't I find a sensor and not have a level (or vice versa)?

Reply:
Ah ok!